setup/src/Magento/Setup/Model/Installer.php

```diff
@@ -773,6 +785,8 @@ public function installSchema()
         $this->setupCoreTables($setup);
         $this->log->log('Schema creation/updates:');
         $this->handleDBSchemaData($setup, 'schema');
+
+        $this->eventManager->trigger(new AfterSchemaWasInstalled($this, $setup, $this->context));
```

Comment:
please, remove $this from the argument. We are return null from the method getTarget. The interface of this class is dirty and we don't want to spread it across the system through event arguments